src/java/org/apache/cassandra/utils/memory/BufferPool.java

```diff
@@ -441,8 +441,8 @@ private Chunk allocateMoreChunks()
                 {
                     if (memoryUsageThreshold > 0)
                     {
-                        noSpamLogger.info("Maximum memory usage reached ({}), cannot allocate chunk of {}",
-                                          readableMemoryUsageThreshold, READABLE_MACRO_CHUNK_SIZE);
+                        noSpamLogger.info("Maximum memory usage reached ({}) for {}, cannot allocate chunk of {}",
```

Comment:
see line 461 , may be " for {} buffer pool " is better and consistent with line 461‘s log message.

Reply:
Thanks for the suggestion. That will provide more context. I will make the change. Do I need to creat a new pull request or make the change based on current one?

Reply:
you can just modify the change on the current code, and doing a new git push to you branch ,then we will see the change here,

Reply:
sounds good. I have pushed the change to my branch.